plugins/inputs/aerospike/aerospike.go

```diff
@@ -89,13 +94,22 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 			}
 		}
 		acc.AddFields("aerospike_node", fields, tags, time.Now())
+		//Finding the latency metrics
+		infoLatency, err := as.RequestNodeInfo(n, "latency:")
```

Comment:
I think it should be possible to get the latency at the same time as the namespaces in a single RequestNodeInfo call, can you look into this as it should be quite a bit more performant.

Reply:
Yaa, We can get the namespace names from the 'latency:' , but from the 'latency:' I got lesser namespace names as compared to 'namespaces'.
<img width="1437" alt="screen shot 2018-03-06 at 11 19 16 am" src="https://user-images.githubusercontent.com/29426724/37016544-c04f2da8-2132-11e8-9b68-2a80094d71c9.png">
Black ones shows the namespaces obtained from 'latency:' whereas the green one is from 'namespaces'.

Reply:
Latency histogram initialization is lazy in nature. It won't exist if we have not done any operation in the namespace. Once we run read or write it will get created and will be shown then on. Hence the namespace names can't be fetched from 'latency:' appropriately.